packages/runtime-core/src/componentEmits.ts

```diff
@@ -100,7 +116,7 @@ export function emit(
     }Modifiers`
     const { number, trim } = props[modifiersKey] || EMPTY_OBJ
     if (trim) {
-      args = rawArgs.map(a => a.trim())
+      args = rawArgs.map((a) => a.trim())
```

Comment:
Prettier is now enforcing parenthesis for arrow functions. Shall we revert to old behavior too (`arrowParens: 'avoid'`)?